src/Pages/RegistrationPage/RegistrationPage.js

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react';
+import CardWrapper from '../../Atoms/Card/Card';
+import InputField from '../../Atoms/InputField/InputField';
+import PriSecBtn from '../../Atoms/PriSecBtn/PriSecBtn';
+import RegistrationPageStyled from './RegistrationPageStyled';
+import { useNavigate } from 'react-router-dom';
+
+const RegistrationPage = () => {
```

Comment:
Maybe form should be created as other component, which will be imported to Registration Page (atomic design --> whole page is made from smaller pieces ).

Reply:
you could think also about some small library for forms: react-hook-form or formik or sth else? 
I'll check it also

Reply:
I think that turning it to smaller component could be a waste of time. Page will appear in the middle - upside nav, downside footer. Inside in my opinion should be a page. not a organism or molecule. But its up to You to decide. This is not a problem.
Im checking a library actually but the form is really small so im doing it just for education purposes. Maybe ill open another test branch to do it in react-hook-form way and then compare differences ?